packages/client/src/clients/in-app-messages/channel-client.ts

```diff
@@ -89,4 +102,22 @@ export class InAppMessagesChannelClient {
       },
     }));
   }
+
+  /*
+   * Socket
+   */
+
+  subscribe(client: InAppMessagesClient) {
```

Comment:
May be worth adding some logs (using `knock`) for potential debugging

Reply:
Yep, will update!